.github/workflows/generator_container_slsa3.yml

```diff
@@ -136,9 +136,9 @@ jobs:
 
           # Generate a predicate only.
           predicate_name="predicate.json"
-          ./"$BUILDER_BINARY" attest --signature="" --predicate="$predicate_name"
-
+          echo $(echo $(echo $UNTRUSTED_DIGEST | cut -d: -f2) $UNTRUSTED_IMAGE) | base64
+          ./"$BUILDER_RELEASE_BINARY" attest --subjects $(echo $(echo $(echo $UNTRUSTED_DIGEST | cut -d: -f2) $UNTRUSTED_IMAGE) | base64 -w0) --predicate="$predicate_name"
```

Comment:
A couple of things.

1. I intentionally set the `--signature=""` option so that this command won't sign the provenance and will just output the SLSA predicate.
2. We don't need to set the subjects here because the `cosign attest` command below will set the subjects for us.

Reply:
We changed to $BUILDER_RELEASE_BINARY according to [this comment](https://github.com/slsa-framework/slsa-github-generator/pull/606#discussion_r927428530) see [the test run from us using $BUILDER_BINARY](https://github.com/Root-DE/Example-Image/runs/7377872538?check_suite_focus=true#step:6:225):
![image](https://user-images.githubusercontent.com/58123348/180400920-cb9313ea-f79a-4a36-b1ee-ddea331a0a83.png)

------------------

When we run this with $BUILDER_RELEASE_BINARY and --signature="" we got the following error:
invalid suffix: "". Must be .intoto.jsonl
This errors originates from attest.go file. According to the file it has to end with .intoto.jsonl, it defaults to attestation.intoto.jsonl, thats why we removed the --signature="".

------------------


Running it without --signature="" we received the error:
expected at least one subject
![image](https://user-images.githubusercontent.com/58123348/180404056-950cfbb8-f550-4d00-ba35-b06be1b0e0d6.png)
you can verify this [here](https://github.com/Root-DE/Example-Image/runs/7378040918?check_suite_focus=true#step:6:225)
Thats why we set the subject in the pr.

Reply:
Ok, I see. This may be a separate bug that we should fix separately. I believe this was caused by #545

Please submit issues when you encounter these errors.

Reply:
Added #614 to track this error.